Short documentation on how to configure CORS

GregDThomas opened the pull request.

dblock commented:
Thanks. Just needs to be linked from FAQ.md I think.

dblock commented on Docs/faq/CORS.md:
```diff
@@ -0,0 +1,96 @@
+## Jetty
+If you are using Jetty, then it's a simple case of using the Jetty 
+[`CrossOriginFilter`](http://www.eclipse.org/jetty/documentation/current/cross-origin-filter.html) to configure CORS. 
+The following `web.xml` fragment gives an example;
```
Maybe a colon or a period instead of a semicolon?

GregDThomas commented:
OK, I added the link from FAQ.md and changed the ";" for a ":". I also updated the example to explicitly include non-default port for one of the servers so that it's clear that is required.

dblock commented:
Merged, thank you.
